tests/fetch-browser.spec.js

```diff
@@ -0,0 +1,36 @@
+var assert = require('assert');
+
+var fetch = require('../build/fetch-browser')();
+
+var good = 'hello world. 你好世界。\n';
+
+function responseToText(response) {
+  if (response.status >= 400) throw new Error('Bad server response');
+  return response.text();
+}
+
+describe('fetch', function() {
+
+  it('should make requests', function (done) {
+    fetch('https://s3.amazonaws.com/bodylabs-edge/unittest/fetch/hello.1.0.0.txt')
```

Comment:
That makes a ton of sense!

Like the node tests, I feel more confident giving these a bigger bracket that effectively covers the entirety of `whatwg-fetch`. I don't have a principled objection to using sinon XHR though, which seems like it would have the same effect.

If you want to go that route, would you mind following this up with that?